### ISSUE TITLE
Placeholder color contrast is less than 4.5:1 

### ISSUE THREAD

**Issue by wghglory** [vmware/clarity#6140]
## Describe the bug

We're using Clarity Input Component, however, the placeholder contrast doesn't meet VPAT accessibility requirement.

![image](https://user-images.githubusercontent.com/11544273/125228833-18306380-e308-11eb-98c6-6c95e56e2333.png)

## Expected behavior

Color contrast is at least 4.5:1 

## Versions

**Clarity project:**

- [ ] Clarity Core
- [x] Clarity Angular/UI

**Clarity version:**

- [x] v3.x
- [x] v4.x
- [x] v5.x

**Framework:**

- [x] Angular
- [ ] React
- [ ] Vue
- [ ] Other:

**Framework version:**
Angular 11

**Device:**

- Type: MacBook
- OS: MacOS 11.5.7
- Browser [chrome, safari]

**Comment by Jinnie** [comment]
Clarity cares deeply about accessibility, and we work with our accessibility team to identify and triage issues. We have an internal tracking system to identify and verify accessibility issues, which has captured this already.
